examples/validate-fee/modules/data_l1/src/main/resources/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %highlight(%-5level) %cyan(%logger{15}) - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
+        <appender-ref ref="STDOUT" />
+    </root>
+
+    <logger name="com.my.validate_fee" level="INFO" />
```

Comment:
Could you add the exceptions to block http4s messages here as in Tessellation? I spun up this example and was able to send the two transactions (seemingly successfully) but I can't really tell because the app logs are inundated with the messages from `ResponseLogger, RequestLogger, etc.`

Reply:
done